server/controllers/message.controller.js

```diff
@@ -177,7 +178,32 @@ function list(req, res) {
     }
 }
 
-function count() {}
+/**
+ * Return count and message IDs of all messages belonging to owner.
```

Comment:
Was there a reason we decided to send back message IDs? When we discussed, I thought we agreed to just send a number. This API endpoint should only be doing a count. This is a much faster operation than getting all of the IDs as well.

Reply:
Yes, we had a discussion about it. Refer to your first comment `In both cases, we would like the count, and possibly an array of message IDs as well` on the ticket https://jira.amida-tech.com/browse/SER-33.